datadog-logs-oci-orm/modules/resourcediscovery/variables.tf

```diff
@@ -0,0 +1,14 @@
+variable "group_id" {
```

Comment:
I'm not sure if "service_id" or "service_name"? might be a better name for this? I see "group_id" as an OCI Group's OCID (eg. `DatadogAuthGroup`)